docs/tutorial/using-checkbox/test-report.rst

```diff
@@ -0,0 +1,87 @@
+.. _test-report:
+
+Review Test Report 
+======================
+
+Checkbox optionally generates test reports in different formats that can be used to easily share the results of a test session. Generally, a :ref:`text summary<test-summary>` and :ref:`submission files<submission-files>` would be created. In this section, you will learn where to find the report files and what they contain.
```

Comment:
I would rephrase this as: Checkbox generates test reports [...]. By default, a test-summary and the submission files are created.

Reply:
"text summary" is the right wording, maybe the reference can be renamed to `text-summary` instead :)

Otherwise, I agree with Max, let's talk about the default behavior of Checkbox (the customized behavior is discussed about in the launcher section anyway).

Reply:
fixed in [fdd5e20](https://github.com/canonical/checkbox/pull/637/commits/fdd5e208e18dfabfacf2c8bf416d1b017b73251b)